### ISSUE TITLE
Android/Phone: Search bar keeps focusing automatically when switching tabs

### ISSUE THREAD

**Issue by andreynering** [brave/brave-browser#20284]
## Description 

Often when I switch tabs, the search bar is focused (and the keyboard opens) and I have to hit "back" to close it constantly.

I find this annoying. If I wanted to switch to another site, I'd touch on the search bar myself.

https://user-images.githubusercontent.com/7011819/147859007-009b8fb0-da80-45e3-b2b0-9fe7eff44ab0.mp4

## Steps to reproduce 

   1. Open tab menu
   2. Click on a tab that was previously open
   3. Instead of just switching to the tab, it'll also focus on the search bar and open the keyboard

## Issue reproduces how often 

Does not seem to happen when you switch to the tab for the first time, but happens when you back to a previous seem tab.

## Version/Channel Information:

Stable release from Google Play store.

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone (does not happen on my tablet)
   - Android version: 8
   
## Brave version

v1.33.106

**Comment by codey-badger** [comment]
Hi @andreynering , it's been a long time but could you confirm if the search bar gets focused (and the keyboard opens) when opening a NEW tab?

**Comment by andreynering** [comment]
@codey-badger Sorry, but I don't use Brave anymore.

**Comment by bsclifton** [comment]
I haven't seen this behavior in a long time; going to close as Stale

@codey-badger opening a new tab doesn't focus search bar and open keyboard as far as I know - you need to manually focus search bar. Does that match what you're experiencing?

**Comment by codey-badger** [comment]
@bsclifton yes, it matches but I need help to know how to change it by myself (to avoid open another issue). I want the behavior to be like other browsers eg Firefox that as soon as I open a new tab the search bar gets focused and the keyboard opens.

**Comment by bsclifton** [comment]
@codey-badger let's open a new issue please - since this was a bug issue capturing problem where search/keyboard showed up

**Comment by codey-badger** [comment]
> [@codey-badger](https://github.com/codey-badger) let's open a new issue please - since this was a bug issue capturing problem where search/keyboard showed up

Not necessary, I found an already opened issue: #40060 
Thanks anyway.
